buildshiprun/handler.go

```diff
@@ -108,24 +105,27 @@ func Handle(req []byte) string {
 			Network: "func_functions",
 			Labels: map[string]string{
```

Comment:
We will need to start storing some of the data in annotations - this is a new feature coming in via faas-netes / faas-swarm etc. IT allows for any value including a date/time stamp in human-readable format.